example/README.md

```diff
@@ -1,74 +1,81 @@
+
+```
+docker run -d -e \
+  COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
+  -p 5775:5775/udp \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 14268:14268 \
+  -p 9411:9411 \
+  jaegertracing/all-in-one:latest
 ```
 
-Second window:
+You should be able to see a web interface by browsing to http://localhost:16686/search
 
-``` 
-> export FLASK_APP=client.py   
-> flask run --port=0 
+![traced request](https://raw.githubusercontent.com/opentracing-contrib/python-flask/example/example/img/jaeger_0.png)
+
+Now, open two terminals and navigate to this directory. Run the following commands in
+each terminal:
+
+First window:
+
+```
+> python3 server.py   
 ```
 
-To test the traces, check what port the client is running on and load
-localhost:port. If you have a lightstep tracer token, you
-should be able to view the trace data on the lightstep interface. 
-(NOTE: if you wish to use a different OpenTracing tracer, simply replace
-`ls_tracer` with the OpenTracing tracer instance of your choice.)
+Give it few seconds to start and run this in second window:
 
-### Trace a request:
+```
+> python3 client.py   
+```
 
-Navigate to `/request/simple/<int:numrequests>` where numrequests is the number
-of requests you want to send to the page. This will send a request to the server
-app, and the trace will include a span on both the client and server sides. This
-occurs automatically since both the client and server functions are decorated
-with @tracer.trace().
+To see the traced requests, go to Jaeger web interface.
+Select your service name from dropdown on the left (it's
+"jaeger_opentracing_example" in our case) and press Find traces button at the bottom of the page.
 
-Result for `/request/simple/1`:
 
-![request and response spans](https://raw.githubusercontent.com/kcamenzind/flask_opentracing/master/example/img/simple.png)
+![traced request](https://raw.githubusercontent.com/opentracing-contrib/python-flask/example/example/img/jaeger.png)
 
-### Log something to a request:
 
-Navigate to `/log`. This will log a message to the server-side span. The client
-span is created automatically through the @tracer.trace() decorator, while the
-logging occurs within the function by accessing the current span as follows:
+(NOTE: if you wish to use a different OpenTracing tracer instead of Jaeger, simply replace
+`jaeger_tracer` with the OpenTracing tracer instance of your choice.)
 
-```python
-span = tracer.get_span()
-span.log_event("hello world")
-```
-Result for `/log`:
+### Trace a request from browser:
 
-![span with log](https://raw.githubusercontent.com/kcamenzind/flask_opentracing/master/example/img/log.png)
+Browse to http://localhost:5000/log and compare the trace in Jaeger.
+The last one has 2 spans instead of 3. The span of webserver's GET method is missing.
+That is because in client.py we pass already existing span over the wire, whereas the request from your browser has to tracing data in it.
```

Comment:
That is because `client.py` starts a trace and passes trace context over the wire, whereas the request from your browser has to tracing context in it.